quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -420,6 +420,16 @@ public void onParticipantConnected(Room room, Participant participant) {
             public void onParticipantDisconnected(Room room, Participant participant) {
                 removeParticipant(participant);
             }
+
+            @Override
+            public void onRecordingStarted(Room room) {
+
```

Comment:
Add a comment to state that this functionality is not available yet but will be in a future release once recording is available in our backend/console.